### ISSUE TITLE
Incorrect aspect ratio displayed for thumbnails

### ISSUE THREAD

**Issue by markgllin** [bpatrik/pigallery2#184]
**Describe the bug**
I've been using the docker nightly build and have noticed recently that the incorrect aspect ratio is being used for thumbnails. I switched over from the `latest` docker build but noticed it had memory issues but aspect ratio was displayed correctly with that version.

Unfortunately, I don't have any particular reproduction steps, it just seems to happen with most of my pictures (although some are displayed correctly for some reason)

**Photo/video (optional) that causes the bug**
An example photo:
![20171223_122352](https://user-images.githubusercontent.com/10660708/93292995-28d58300-f7a4-11ea-9953-1061b9bd8a37.jpg)

**Screenshots**
However, the thumbnail is displayed with equal height and width
<img width="392" alt="image" src="https://user-images.githubusercontent.com/10660708/93293053-4c003280-f7a4-11ea-9035-5fe68d4aa483.png">

**Environment (please complete the following information):**
- I'm currently running PiGallery on a Raspberry Pi 4, 64-bit enabled within docker
 - I've been able to repro on both Chrome and Firefox (unable to load on safari)

**Used app version**:
- docker-nightly (specifically alpine)

Happy to provide any more details/logs I can. As far as I can tell, no errors appeared in the logs during thumbnail creation or indexing

**Comment by markgllin** [comment]
looking through past issues, this looks like it's related to https://github.com/bpatrik/pigallery2/issues/172 and https://github.com/bpatrik/pigallery2/issues/19

**Comment by bpatrik** [comment]
yes for some reason I could not figure out why these photos are rotated. I'll try to give an other look with your photo.

**Comment by lbischof** [comment]
This also happens for me on the nightly build. The same picture (thumbnail) is shown correctly in the album art though.

**Comment by bpatrik** [comment]
Yes its a new issue. With 1.8.0 the error would occure once you zoom in.  Could not find the source of it yet.

**Comment by lbischof** [comment]
I'm not sure what you mean with "rotated". I was able to fix this behavior by adding `object-fit: cover` to the `img`. This also seems to be relatively well supported: https://caniuse.com/object-fit.

**Comment by bpatrik** [comment]
Thanks for the PR, but I'm not sure that it will fix the issue.

The issue is the following:
* Some of the photos are rotated in the bitmap, but the exif data signals that rotation ([more here](https://www.impulseadventure.com/photo/exif-orientation.html)). Since recently, browser can handle it and based on the exif data rotate it back. 
* Grid is rendered based on the information that the server can read from the photo. It can read the original (bitmap) width and height and the info in the exif (exif width+ height  + orientation)
* Some of the photos (seems to be related to Iphone) are correctly rotated in the bitmap, but exif data suggest that their orientation is still not correct. Others photos are not correctly rotated and exif data also suggest their orientation is not correct.

I tried to resolve this, bit on the server side I could not figure out in which case should I rotate the dimension(width and height) of the photo and in which case not. I found that taking the orientation data into account produces more valid results than the other way around.

This is the code in question:
https://github.com/bpatrik/pigallery2/blob/8c2654980d74cc8735c1ed965fa981f25bfca04e/src/backend/model/threading/MetadataLoader.ts#L88


I think setting `object-fit:cover` will result in some cropping, isn't it?
